src/realm/metrics/query_info.cpp

```diff
@@ -34,7 +34,11 @@ QueryInfo::QueryInfo(const Query* query, QueryType type)
     const Group* group = query->m_table->get_parent_group();
     REALM_ASSERT(group);
 
-    m_description = query->get_description();
+    try {
+        m_description = query->get_description();
+    } catch (const SerialisationError& e) {
```

Comment:
Is a SerialisationError the only error that can be thrown here?

Reply:
It's the only "expected" error. I suppose there could also be unexpected errors such as running out of memory while constructing the description string. But in that case I think it is ok to fail, since we would most likely fail immediately afterwards anyways.